coalib/settings/Section.py

```diff
@@ -161,6 +200,11 @@ def __getitem__(self, item, ignore_defaults=False):
 
         res = self.contents.get(key, None)
         if res is not None:
+            if res.to_append and self.defaults and (res.key in self.defaults):
+                return Setting(
+                    key=key,
+                    value=self.defaults[key].value + ',' + res.value,
```

Comment:
let's have a space after the comma for beautification, shall we? :) Not that it's functionally relevant though.